scripts/bump_extension_version.sh

```diff
@@ -0,0 +1,31 @@
+###
+# Bump the extension version.
+#
+# Usage:
+#   ./scripts/bump_extension_version.sh 2023.30.0 2023.32.0
+###
+
+set -euxo pipefail
+
+FROM=$1
+TO=$2
+
+# Create a branch.
+git checkout -B v$TO main
+
+# Update the version in-place.
+rg $FROM --files-with-matches | xargs sed -i "" "s/$FROM/$TO/g"
+
+# Re-lock dependencies.
+rm requirements.txt
+rm requirements-dev.txt
+pip-compile --generate-hashes --resolver=backtracking -o ./requirements.txt ./pyproject.toml
+pip-compile --generate-hashes --resolver=backtracking --upgrade --extra dev -o ./requirements-dev.txt ./pyproject.toml
+npm install --package-lock-only
+
+# Commit the change.
+git add .
+git commit -m "Bump extension version to $TO"
```

Comment:
nit: I'd use `git commit -am` instead of `git add .` unless you _want_ to add untracked files to the commit. Otherwise it seems too easy to include files by accident.

Reply:
Thanks!